native-messaging/app/ping_pong_win.bat

```diff
@@ -1,3 +1,3 @@
 @echo off
 
-call python C:\path\to\ping_pong.py
+call python C:\\Users\\andy\\sandboxes\\webextensions-examples\\native-messaging\\app\\ping_pong.py
```

Comment:
I think this change is unintentional too?

Reply:
Umm yeah, I'll remove those changes.